documentation/doc_sncf/source/integration.rst

```diff
@@ -1056,6 +909,11 @@ name             string                          Name of the commercial mode
 physical_modes   array of `physical_mode`_       Physical modes of this commercial mode
 ================ =============================== =======================================
 
++-----------------------------------------------------------------------------------------------------------+
+| *Note*                                                                                                    |
+| As commercial mode you will find in the API SNCF (TGV, TER, Intercité, Transilien)                                                                                                         |   
```

Comment:
you mean to list the commercial modes here ?

If so I think you should maybe be more explicit:

```
The commercial modes available in the SNCF API are:
 * ...
 * ...
```